src/firmware_new/boards/frankie_v1/constants.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+// This file contains all constants that are shared between our software (AI)
+// and firmware code. Since this needs to be compiled by both C and C++, everything
+// should be defined in a way that's compatible with C.
+//
+// NOTE: This file is here until its moved to the shared folder after the new firmware
+// is setup
+//
+
+// the IPv6 multicast address, only ff02 is important, the rest is random
+const char* AI_MULTICAST_ADDRESS = "ff02::c3d0:42d2:bb8";
+// the port to listen to the multicast packets for the robot (multicast)
+const unsigned ROBOT_MULTICAST_LISTEN_PORT = 42000;
+// the port that the AI is listening on for robot msgs (unicast)
```

Comment:
since this comment is duplicated below for the `AI_UNICAST_LISTENPORT`, this once should probably say something like
"the port to send unicast packets to for robot msgs"

Reply:
fixed

Reply:
think you got it backwards. `ROBOT_MULTICAST_LISTEN_PORT` should have the new comment you made (just flip them).

Reply:
those are equal to each other, so its technically still correct, but I flipped it now